app/src/main/java/org/oppia/android/app/profile/AddProfileViewModel.kt

```diff
@@ -18,6 +18,7 @@ class AddProfileViewModel @Inject constructor() : ObservableViewModel() {
   val createPin = ObservableField(false)
   val isButtonActive = ObservableField(false)
   val showInfoAlertPopup = ObservableField<Boolean>(false)
+  val starRequired: String = "*Required"
```

Comment:
Here `*` should be mentioned here by `required` should come from `strings.xml` file.

Also whatever error you are getting can you mention it here or create a [debugging doc](https://github.com/oppia/oppia/wiki/Debugging-Docs) mentioning what steps you performed and what errors you got.